### PR TITLE
Drop Bintray references from `README.md`

### PR DESCRIPTION
**Pull request by Stephan202**
Analogous to 86a1cd350407d623d910c1d5c6b24e1ba9bba38b.

### CODE REVIEW

**Comment by palantirtech** [comment]
Thanks for your interest in [palantir/gradle-baseline](https://github.com/palantir/gradle-baseline), @Stephan202! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

File: README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ buildscript {
 }
 
 repositories {
-    maven { url  "http://palantir.bintray.com/releases" }
+    mavenCentral()
```
Comment (Stephan202) [inline README.md:51]: I followed the approach taken in #1728, but I am not myself a Gradle user, so perhaps this is the default configuration, in which case this section could be dropped :shrug:.

**Review by Stephan202 — COMMENTED** [review]
(I'm using the Error Prone checks in a Maven project and previously configured Bintray for that; today I noticed that Bintray started to return 403s. Hence this update.)

**Review by carterkozak — APPROVED** [review]
Thanks!

**Pull request merged**